app/src/main/java/org/mozilla/fenix/components/FenixSnackbar.kt

```diff
@@ -157,8 +162,20 @@ class FenixSnackbarPresenter(
         actionName: String? = null,
         isError: Boolean = false
     ) {
-        FenixSnackbar.make(view, length, isError).setText(text).let {
+        val snackbar = FenixSnackbar.make(view, length, isError).setText(text).let {
             if (action != null && actionName != null) it.setAction(actionName, action) else it
-        }.show()
+        }
+
+        if (view.context.settings().shouldUseBottomToolbar) {
+            snackbar.view.setPadding(
+                0,
```

Comment:
Nit: Is the default padding (used for top toolbar) just 0 for all sides?  Could we make that explicit?